### PR TITLE
[Serializer] Deprecate autowiring aliases pointing to concrete normalizers

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | yes
| Tickets       | -
| License       | MIT
| Doc PR        | -

They should've never been added. See https://github.com/symfony/symfony/pull/22098#discussion_r107220544

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Missing changelog+upgrade entries ;)

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php [hunk 1]
```diff
@@ -127,6 +127,7 @@
             ->tag('serializer.normalizer', ['priority' => -1000])
 
         ->alias(ObjectNormalizer::class, 'serializer.normalizer.object')
+            ->deprecate('symfony/serializer', '6.2', 'The "%alias_id%" service alias is deprecated, use "'.NormalizerInterface::class.'" instead.')
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php]: isn't implementing NormalizerAwareInterface the right replacement in some cases (like the one described in the docs) ?

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php]: indeed, suggestion updated

**Comment by chalasr** [comment]
Fixed!

**Review by dunglas — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Comment by ogizanagi** [comment]
Thanks @chalasr.

**Pull request merged**